src/main/resources/org/opentripplanner/apis/transmodel/schema.graphql

```diff
@@ -2092,6 +2097,58 @@ input PenaltyForStreetMode {
   timePenalty: DoubleFunction!
 }
 
+"""
+One of the listed stop locations must be visited on-board a transit vehicle or the journey must
+alight or board at the location.
+"""
+input PlanPassThroughViaLocationInput {
+  "The label/name of the location. This is pass-through information and is not used in routing."
+  label: String
+  """
+  A list of stop locations. A stop location can be a quay, a stop place, a multimodal
+  stop place or a group of stop places. It is enough to visit ONE of the locations
+  listed.
+  """
+  stopLocationIds: [String!]
+}
+
+"""
+A via-location is used to specifying a location as an intermediate place the router must
+route through. The via-location must be either a pass-through-location or a
+visit-via-location. An on-board "visit" is only allowed for pass-through-via-locations, while
+the visit-via-location can visit a stop-location or a coordinate and specify a
+minimum-wait-time.
+"""
```

Comment:
"An on-board "visit" is only allowed for pass-through-via-locations" this can be interpreted in two ways, either it limits the passthrough point or a visit point. The difference between the two types is not completely clear as it's not clear what visit means and sometimes passthrough point and visit stop location both act similarly as a boarding/alighting location.

Reply:
I removed the unclear part, I think it does not belong hear anyway.

Reply:
We can improve the doc when adding coordinates. It is probably wise to move some of this down onto the coordinates  field/type doc - but since it does not exist yet, I will just keep this as is.